addons/account/views/res_users_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="res_users_form_view" model="ir.ui.view">
```

Comment:
According to the now hundred-year-old guidelines, it should be named res_users_view_form.

About name: res.users.view.form.inherit.account/sale/...